Tasks/VsTestV2/runvstest.ts

```diff
@@ -78,6 +79,39 @@ async function execute() {
     }
 }
 
+function isHydraFlowToBeEnabled(inputDataContract: InputDataContract) {
+    if ((inputDataContract.ServerType && inputDataContract.ServerType.toLowerCase() === 'hosted')) {
+
+        tl.debug('Enabling Hydra flow since serverType is hosted.');
+        return true;
+    }
+
+    if (tl.getVariable('Force_Hydra') && tl.getVariable('Force_Hydra').toLowerCase() === 'true') {
```

Comment:
Is this a hidden /backdoor variable you are proposing?

Reply:
Yes